src/Jackett.Common/Definitions/zooqle.yml

```diff
@@ -72,7 +72,7 @@ settings:
 search:
   # includes only verified torrents
   paths:
-    - path: "search?s={{ .Config.sort }}&v=t&sd={{ .Config.type }}&q={{ if .Keywords }}{{ .Keywords }}{{ else }} *{{ end }}{{ if .Categories }} category:{{ range .Categories }}{{.}},{{end}}{{ else }}{{ end }}"
+    - path: "search?s={{ .Config.sort }}&v=t&sd={{ .Config.type }}&q={{ if .Query.IMDBID }}{{ .Query.IMDBID }}{{ else }}{{ if .Keywords }}{{ .Keywords }}{{ else }} *{{ end }}{{ if .Categories }} category:{{ range .Categories }}{{.}},{{end}}{{ else }}{{ end }}"
```

Comment:
unfortunately this will not work as the if-then-else does not support nesting.
please revert the zooqle change for now, so we can go ahead with the TGx implementation
(in future, please submit a separate PR for each indexer so we can track, implement or reject them individually).
I will do some research to see if I can figure out a way to support imdbid searching on zooqle.

Reply:
Thanks, I will raise another PR after revert the zooqle changes.